### DISCUSSION TITLE
Keyboard shortcut for toggling checkboxes in Task List plugin?

### DISCUSSION THREAD

**Discussion by reagle** [zim-desktop-wiki/zim-desktop-wiki#2044] [Ideas]
Is this already possible, if not, would it be a useful feature request?

**Comment by bruvellu** [comment]
It's already possible:

- **F12** toggles mark as completed (v)
- **SHIFT+F12** toggles mark as canceled (x)

To change these or add shortcuts to other checkbox actions go to `Edit > Preferences > Key bindings`

**Comment by reagle** [comment]
Thank you! This should be documented on the help page: https://zim-wiki.org/manual/Plugins/Task_List.html
